sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorClient/AnomalyDetectionLiveTests.cs

```diff
@@ -251,7 +251,7 @@ await foreach (IncidentRootCause rootCause in client.GetIncidentRootCausesAsync(
         [RecordedTest]
         public async Task GetIncidentRootCausesForIncidentFromAlert()
         {
-            const string incidentId = "5a0692283edccf37ce825b3a8d475f4e-17571a77000";
+            const string incidentId = "30612c95b4c216ef418956c5c6162691-17bbd8dec00";
```

Comment:
We're manually setting the incident ID here because the `IncidentId` we use for other tests does not return the information we want to validate here.